rwviewer/ViewerWidget.cpp

```diff
@@ -163,7 +165,11 @@ void ViewerWidget::drawFrameWidget(ModelFrame* f, const glm::mat4& m) {
         glLineWidth(1.f);
     }
     dp.textures = {whiteTex};
-    renderer->getRenderer()->drawArrays(thisM, _frameWidgetDraw, dp);
+    
+    RW_CHECK(renderer != nullptr, "GameRenderer is null");
+    if(renderer != nullptr) {
+        renderer->getRenderer()->drawArrays(thisM, _frameWidgetDraw, dp);
+    }
```

Comment:
agreed, but that's a seperate issue.
(We are currently transitioning renderers, first internally and then to librw by aap, that broke a ton of stuff)